Add .bowerrc to the NuGet package,

tomalec opened the pull request:
to make `bower link clientPackage` work out of the box for Starcounter installation folder

tomalec committed:
Add .bowerrc to the NuGet package,

to make `bower link clientPackage` work out of the box for Starcounter installation folder

warpech commented on CONTRIBUTING.md:
```diff
@@ -1,10 +1,20 @@
+## Development flow tips
+To slightly help with the "repolepsy" and make working with multiple repos and elements together easier, you may consider using [`bower link`](https://bower.io/docs/api/#link).
+- an keep the clone of some client package repo in a separate folder, for example `/path/to/imported-template`,
```
```suggestion
 - keep the clone of some client package repo in a separate folder, for example `/path/to/imported-template`,
````

warpech reviewed (approved):
Lgtm! Minor thing:

warpech committed:
Add editorial change CONTRIBUTING.md

as suggested at https://github.com/Starcounter/StarcounterClientFiles/pull/127#discussion_r235174888

Co-Authored-By: tomalec <tomalecpub@gmail.com>
